project/BuildSettings.scala

```diff
@@ -47,12 +47,12 @@ object BuildSettings extends AutoPlugin {
         "-deprecation", // Emit warning and location for usages of deprecated APIs.
         "-feature", // Emit warning and location for usages of features that should be imported explicitly.
         "-unchecked", // Enable additional warnings where generated code depends on assumptions.
-        "-Xfatal-warnings", // Fail the compilation if there are any warnings.
+        //"-Xfatal-warnings", // Fail the compilation if there are any warnings.
```

Comment:
Is there any reason to disable these compiler rules?

Reply:
Please see BuildSettings.scala L18-39 that now handles scalac 2.12 and 2.13 options much more elegantly.